Packs/MicrosoftGraphUser/TestPlaybooks/Microsoft_Graph_User_test.yml

```diff
@@ -310,22 +305,21 @@ tasks:
     quietmode: 0
   "9":
     id: "9"
-    taskid: 5e97172f-3717-4fe5-8f4f-8beb05667f94
+    taskid: 5ace3b5a-c0b2-4221-8611-90a4ac358ed9
     type: title
     task:
-      elasticcommonfields: {}
-      id: 5e97172f-3717-4fe5-8f4f-8beb05667f94
+      id: 5ace3b5a-c0b2-4221-8611-90a4ac358ed9
       version: -1
-      name: Done!
+      name: Error!
```

Comment:
if you consider this an error, need to return an error entry, as currently the test will pass

Reply:
@Itay4 I just added the two 'PrintErrorEntry' tasks to test playbook. 
Please not that this step is currently not mentioned on the dev [page](https://xsoar.pan.dev/docs/integrations/test-playbooks).